internal/provider/provider.go

```diff
@@ -69,18 +69,33 @@ func (p *btpcliProvider) Schema(_ context.Context, _ provider.SchemaRequest, res
 				MarkdownDescription: "The identity provider to be used for authentication (default: SAP ID service with origin `sap.default`).",
 				Optional:            true,
 			},
+			"idp_url": schema.StringAttribute{
```

Comment:
Due to the large amount of parameters and combinations, we should add stringvalidators here as we did for the idtoken, to make sure that the combinations are validated.

- For the three new fields the `alsoRequires` should be set
- The `idtoken` validation `tringvalidator.ConflictsWith` should be enhanced with the three new fields